Project.toml

```diff
@@ -23,17 +23,14 @@ julia = "1.0"
 [extras]
 BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
 CSVFiles = "5d742f6a-9f54-50ce-8119-2520741973ca"
+CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
```

Comment:
Here it says that CodecZlib was added in this PR.

Reply:
CodecZlib is for TranscodingStreaming or something, no idea why it is there, a quick look at the documentation shows it may be used for compression, so this would not be part of my PR. It looks like compared to my Project.toml file, the package just got moved earlier in the [extras] list compared to what DrWatson 2.4.0 has.

Reply:
Ooooh sorry about that my fault. Yeap it is added in another PR, I forgot... apologies.